hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestReconTaskControllerImpl.java

```diff
@@ -161,17 +179,35 @@ public void testBadBehavedTaskBlacklisting() throws Exception {
   @Test
   public void testReInitializeTasks() throws Exception {
 
-    OMMetadataManager omMetadataManagerMock = mock(OMMetadataManager.class);
+    ReconOMMetadataManager omMetadataManagerMock = mock(
+        ReconOMMetadataManager.class);
     ReconDBUpdateTask reconDBUpdateTaskMock =
         getMockTask("MockTask2");
     when(reconDBUpdateTaskMock.reprocess(omMetadataManagerMock))
         .thenReturn(new ImmutablePair<>("MockTask2", true));
+    when(omMetadataManagerMock.getLastSequenceNumberFromDB()
+    ).thenReturn(100L);
 
+    long startTime = System.currentTimeMillis();
     reconTaskController.registerTask(reconDBUpdateTaskMock);
     reconTaskController.reInitializeTasks(omMetadataManagerMock);
+    long endTime = System.currentTimeMillis();
 
     verify(reconDBUpdateTaskMock, times(1))
         .reprocess(omMetadataManagerMock);
+
+    verify(omMetadataManagerMock, times(1)
+    ).getLastSequenceNumberFromDB();
+
+    ReconTaskStatusDao dao = new ReconTaskStatusDao(sqlConfiguration);
+    ReconTaskStatus reconTaskStatus = dao.findById("MockTask2");
+    long taskTimeStamp = reconTaskStatus.getLastUpdatedTimestamp();
+    long seqNumber = reconTaskStatus.getLastUpdatedSeqNumber();
+
+    Assert.assertTrue(startTime <= taskTimeStamp
```

Comment:
These assertions for mocked values are meaningless right, instead, making sure mocks are verified should suffice.

Reply:
These assertions are on the actual dao.